modules/core/src/test/java/org/apache/ignite/internal/processors/database/IgniteDbPutGetAbstractTest.java

```diff
@@ -68,6 +80,24 @@ private IgniteEx ig() {
         return grid(0);
     }
 
+    /** {@inheritDoc} */
+    @Override protected IgniteConfiguration getConfiguration(String gridName) throws Exception {
+        IgniteConfiguration cfg = super.getConfiguration(gridName);
+
+        if (getTestIgniteInstanceIndex(gridName) == smallWalBufSizeNodeIdx) {
+            cfg.getDataStorageConfiguration()
+                .setWalBufferSize(DFLT_WAL_SEGMENT_SIZE / 4)
+                .setWalSegmentSize(DFLT_WAL_SEGMENT_SIZE / 4);
+        }
+
+        cfg.setFailureHandler((ignite, failureCtx) -> {
```

Comment:
Don't we have `getFailureHandler()` in abstract test? This section can be moved to that specific method.

Reply:
Nice catch! Will change of course.